src/renderer/components/settings/Information.tsx

```diff
@@ -44,10 +44,12 @@ export class LanguageSettings extends React.Component<Props, States> {
 
     public async componentDidMount() {
         const { locale } = this.props;
-        let folderPath: string = path.join((global as any).__dirname, "assets/md/information");
+        const infoFolderRelativePath = "assets/md/information";
+
+        let folderPath: string = path.join((global as any).__dirname, infoFolderRelativePath);
         try {
             if (__PACKAGING__ === "0") {
-                folderPath = path.join(process.cwd(), "dist/assets/md/information");
+                folderPath = path.join(process.cwd(), infoFolderRelativePath);
```

Comment:
Shouldn't this be
```
`dist/${infoFolderRelativePath}`
```
?

Reply:
Or better even:
`folderPath = path.join(process.cwd(), "dist", "infoFolderRelativePath");`

Reply:
Please use the multi-param `path.join()` variant, as this will take care of forward/reverse slash path separator.

Reply:
done

Reply:
done